src/components/card/index.js

```diff
@@ -170,29 +161,32 @@ class Card extends BaseComponent {
     const blurOptions = this.getBlurOptions();
     const Container = onPress ? TouchableOpacity : View;
     const brRadius = borderRadius || DEFAULT_BORDER_RADIUS;
+    const animationProps = this.extractAnimationProps();
     
     return (
-      <Container
-        style={[
-          this.styles.container,
-          {borderRadius: brRadius},
-          this.elevationStyle,
-          this.shadowStyle,
-          this.blurBgStyle,
-          containerStyle,
-          style,
-        ]}
-        onPress={onPress}
-        delayPressIn={10}
-        activeOpacity={0.6}
-        testID={testID}
-        {...others}
-      >
-        {Constants.isIOS && enableBlur && <BlurView style={[this.styles.blurView, {borderRadius: brRadius}]} {...blurOptions}/>}
-        <View width={width} height={height} row={row} style={[this.styles.innerContainer, {borderRadius: brRadius}]}>
-          {this.renderChildren()}
-        </View>
-      </Container>
+      <Animatable.View {...animationProps}>
```

Comment:
I thought we've decided on letting the developer wrap the components instead of us doing it for him.
If we do decide to keep it, I personally think we should condition this, otherwise, it creates an unnecessary hierarchy for our components (in most of the cases)

Reply:
Condition added.